src/components/SecondaryNav/SecondaryNav.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import className from 'classnames';
+import './SecondaryNav.css';
+
+
+/**
+ * A navigation item to be used in the secondary nav bar
+ *
+ */
+export const NavLink = ({ active, children }) => {
+
+  const linkClass = className({
+    'SecondaryNav--link': true,
+    'SecondaryNav--link-active': active,
+  });
+
+  return (
+    <li className={linkClass}>{children}</li>
+  );
+}
+
+NavLink.propTypes = {
+  /** Child elements to display on the navbar */
+  children: propTypes.node,
+  /** Whether the link is active */
+  active: propTypes.bool,
+};
+
+NavLink.defaultProps = {
+  children: null,
+  active: false,
+};
+
+
+/**
+ * Secondary navigation typically used under the header
+ *
+ * Must pass an array of NavLinks for the navigation bar.
+ * Each NavLink should contain an anchor with appropriate href, or a react
+ * router Link or equivelant.
+ */
+const SecondaryNav = ({ buttons }) => {
+
+  const navClass = className(
+    'SecondaryNav--navigation',
```

Comment:
@dankolbman did we decide that this needed to have `--navigation`, seems redundant? We need to add better docs for style naming into the queque